src/tests/ftest/cart/corpc/cart_corpc_five_node.py

```diff
@@ -46,6 +46,11 @@ def setUp(self):
         self.utils = CartUtils()
         self.env = self.utils.get_env(self)
 
+    def tearDown(self):
+        """ Tear down """
+        super(CartCoRpcFiveNodeTest, self).tearDown()
+        self.utils.cleanup_processes()
+
```

Comment:
This will not completely solve the problem.   If the test encounters any other teardown errors or an internal cancel call is ever added the `super(CartCoRpcFiveNodeTest, self).tearDown()` call will end up exiting the test (either [here ](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/apricot/apricot/test.py#L271)or [here](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/apricot/apricot/test.py#L276)) before `self.utils.cleanup_processes()` can be called.  To avoid this I'd suggest:
```suggestion
    def tearDown(self):
        """ Tear down """
        self.report_timeout()
        self.utils.cleanup_processes()
        super(CartCoRpcFiveNodeTest, self).tearDown()
```

Also as an improvement it would be good to know if the `self.utils.cleanup_processes()` was successful.  After calling `stop_processes(["localhost"], "'(crt_launch|orterun)'")` it would be good to check that a `pgrep '(crt_launch|orterun)'` does not match anything.  To support marking a test with a fail/error state if it is unable to stop the `crt_launch|orterun` processes, the `self.utils.cleanup_processes()` would need to return a list of error messages (empty if there are no errors) and that list would be handled like so:
```
    def tearDown(self):
        """ Tear down """
        self.report_timeout()
        self._teardown_errors.extend(self.utils.cleanup_processes())
        super(CartCoRpcFiveNodeTest, self).tearDown()
```

Reply:
Let's address this in a separate PR.